### PR TITLE
blog: Bringing DLC to Lightning Part 3

### PR DESCRIPTION
**Pull request by holzeis**
resolves https://github.com/get10101/marketing/issues/33

### CODE REVIEW

**Review by luckysori — APPROVED** [review]
Nice work! Thanks for writing this.

File: blog/2023-08-29-bringing-dlc-to-lightning-part-3.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+
+In part 3, we are going to write about how 10101 is utilizing virtual channels to bring DLCs to Lightning.
+
+
+## DLC Channel
+
+In order to bring DLCs to Lightning we have to firstly bring them off-chain.
+That is that we will have to secure our funds even if we do not directly go on-chain after a DLC expired (after the oracle attested to the outcome of an event).
+At this point you may want to setup another contract or simply extend the contract.
```
[the remark this answers is not in the document]
Comment (holzeis) [reply blog/2023-08-29-bringing-dlc-to-lightning-part-3.md:24]: Hmmm.. no actually I wanted to address here that we first have to make a DLC able to live off-chain, without having to go on-chain whenever it is expired. For that we need it to be revocable.

How to link it to a Lightning channel is described in the next section.

**Review by bonomat — COMMENTED** [review]
I'll leave the commenting to Lucas to not bother you with a second opinion :)

File: blog/2023-08-29-bringing-dlc-to-lightning-part-3.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+The follwoing diagram illustrates the complete transaction structure of a DLC enabled Lightning channel.
+Note that different publish and revocation keys are used for the split transaction and buffer transaction.
+
+![](/2023-08-29-bringing-dlc-to-lightning-part-3/split_transaction.png)
+
+## Closing Thoughts
+
+With a dlc channel living alongside the lightning channel we are bringing DLCs to Lightning while still keeping a fully functional Lightning Wallet.
+This allows for a smooth and fast experience without a delay due to on-chain transactions while keeping the same security as a regular Lightning Channel.
```
[the remark this answers is not in the document]
Comment (holzeis) [reply blog/2023-08-29-bringing-dlc-to-lightning-part-3.md:70]: I've removed the sentence, as I don't think it adds any more value.

File: blog/2023-08-29-bringing-dlc-to-lightning-part-3.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (luckysori) [reply blog/2023-08-29-bringing-dlc-to-lightning-part-3.md]: > I wanted to address here that we first have to make a DLC able to live off-chain, without having to go on-chain whenever it is expired. For that we need it to be revocable.

Fair enough, that makes sense. I think this explanation you gave me is clearer!

In the original I struggled with the word "secure".

[the remark this answers is not in the document]
Comment (holzeis) [reply blog/2023-08-29-bringing-dlc-to-lightning-part-3.md]: I've adapted the sentence slightly to make it clearer.

File: static/2023-08-29-bringing-dlc-to-lightning-part-3/dlc_channel.png
[the remark this answers is not in the document]
Comment (holzeis) [reply static/2023-08-29-bringing-dlc-to-lightning-part-3/dlc_channel.png:1]: what is the refund transaction?

[the remark this answers is not in the document]
Comment (luckysori) [reply static/2023-08-29-bringing-dlc-to-lightning-part-3/dlc_channel.png:1]: A transaction that ensures that the buffer transaction is spendable even if the oracle never attests to anything. After an absolute timelock past the attestation time, it spends the DLC back to the two parties in the original proportion.

**Pull request merged**